projects/Generic/options

```diff
@@ -46,6 +47,7 @@
 
   # Project CFLAGS
     PROJECT_CFLAGS="-mmmx -msse -msse2 -mfpmath=sse"
+    TARGET_FEATURES+=" mmx sse sse2"
```

Comment:
Is the whitespace in front of mmx necessary?

Reply:
Yes, because of string string concatenation. Without space in front you would get _64bitmmx_ feature.

Reply:
Ahhh right missed that :)